### DISCUSSION TITLE
呃呃呃似乎出错了

### DISCUSSION THREAD

**Discussion by C6H8FNO4** [BeyondDimension/SteamTools#3372] [General]
![Uploading 屏幕截图 2024-06-28 002819.png…]()

**Comment by pjy612** [comment]
图没传好

**Comment by C6H8FNO4** [comment]
![屏幕截图 2024-06-28 002819](https://github.com/BeyondDimension/SteamTools/assets/75513950/37a812f1-7803-4f01-952f-cdc0e8cd499f)

**Comment by pjy612** [comment]
beta 版 steam ？可能不识别
